controllers/cloud.redhat.com/clowdapp_controller.go

```diff
@@ -164,7 +159,7 @@ func (r *ClowdAppReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 	if err != nil {
 		r.Recorder.Eventf(&app, "Warning", "ClowdEnvMissing", "Clowder Environment [%s] is missing", app.Spec.EnvName)
 		SetClowdAppConditions(ctx, r.Client, &app, crd.ReconciliationFailed, err)
-		return ctrl.Result{Requeue: true}, err
+		return ctrl.Result{Requeue: true, RequeueAfter: time.Second * 10}, err
```

Comment:
Why have you decided to start requeueing after 10s now?

Reply:
Just to slow things down a little and give a little more time for things to "recover"/steady state

Reply:
I understand this may cut down on failed reconciliations, but is that worth the likely slow down from getting an environment reconciled?

Reply:
@kylape IMO yes it is.  We need stability and if we take a perf hit so be it, we can always optimize later when things have calmed down.  Happy to open a jira for the optimization if we want it in the future.

Reply:
Pete and I dsicussed this, and we decided to compromise by reducing the initial requeue delay to 2s.